atomic_defi_design/qml/Exchange/Trade/SimpleView/SubBestOrder.qml

```diff
@@ -0,0 +1,199 @@
+            anchors.fill: parent
+            RowLayout                           // Order Token
+            {
+                property int _iconWidth: 24
+
+                Layout.preferredWidth: _tokenColumnSize
+                DefaultImage                         // Order Token Icon
+                {
+                    Layout.preferredWidth: parent._iconWidth
+                    Layout.preferredHeight: 24
+                    source: General.coinIcon(coin)
+                }
+                DefaultText                          // Order Token Name
+                {
+                    id: _tokenName
+                    Layout.preferredWidth: _tokenColumnSize - parent._iconWidth
+                    text: coin
+                    font.pixelSize: 14
+                }
+            }
+            
+            DefaultText                         // Order Available Quantity
+            {
+                Layout.preferredWidth: _quantityColumnSize
+                text: parseFloat(General.formatDouble(quantity, General.amountPrecision, true)).toFixed(8)
+                font.pixelSize: 14
+            }
+            
+            DefaultText                         // Order Available Quantity In BASE
+            {
+                Layout.preferredWidth: _quantityInBaseColumnSize
+                text: parseFloat(General.formatDouble(base_max_volume, General.amountPrecision, true)).toFixed(8)
+                font.pixelSize: 14
+            }
+            
+            DefaultText                         // Order Fiat Volume
+            {
+                Layout.preferredWidth: _fiatVolumeColumnSize
+                text: price_fiat+API.app.settings_pg.current_fiat_sign
+            }
+            
+            DefaultText
+            {
+                Layout.preferredWidth: _cexRateColumnSize
+                color: cex_rates=== "0" ? Qt.darker(theme.foregroundColor) : parseFloat(cex_rates)>0? theme.redColor : theme.greenColor
+                text: cex_rates=== "0" ? "N/A" : parseFloat(cex_rates)>0? "+"+parseFloat(cex_rates).toFixed(2)+"%" : parseFloat(cex_rates).toFixed(2)+"%"
+            }
+            DefaultTooltip
+            {
+                id: _tooltip
+                dim: true
+                modal: true
+                closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutside
+                width: 250
+                contentItem: DexLabelUnlinked
+                {
+                    text_value: qsTr(" %1 is not enabled - Do you want to enable it to be able to select %2 best orders ?<br><a href='#'>Yes</a> - <a href='#no'>No</a>").arg(coin).arg(coin)
+                    wrapMode: DefaultText.Wrap
+                    width: 250
+                    onLinkActivated:
+                    {
+                        if (link==="#no") _tooltip.close()
+                        else
+                        {
+                            if (API.app.enable_coins([coin]) === true) _tooltip.close()
```

Comment:
`API.app.enable_coin(coin)` also exists if you do not want to create yourself an array.